docs/azure_jumpstart_arcbox/_index.md

```diff
@@ -208,7 +208,7 @@ ArcBox must be deployed to one of the following regions. Deploying ArcBox outsid
 After deployment is complete, its time to start exploring ArcBox. Most interactions with ArcBox will take place either from Azure itself (Azure Portal, CLI or similar) or from inside the ArcBox-Client virtual machine. When remoted into the client VM, here are some things to try:
 
 * Open Hyper-V and access the Azure Arc-enabled servers
-  * Username: arcdemo
+  * Username: ArcBox-Ubuntu (arcdemo), ArcBox-SQL/ArcBox-Win (Administrator) 
```

Comment:
I like this idea but I think a table showing usernames for each machine might be easier to follow.

```suggestion
  * Username (for Windows servers): Administrator
  * Username (for Linux servers): arcdemo
```